### ISSUE TITLE
Strapi model id's are numbers while in this tool its TS output id's are typed as string

### ISSUE THREAD

**Issue by timohofmeijer** [erikvullings/strapi-to-typescript#11]
This tool is generating TS interfaces where the model id's are of type `string` while in the Strapi api response they are numbers. This forces me to typecast either of them before I can work with it. I this a known thing?

**Comment by aperron** [comment]
hello,

The field id was hard coded.
I released a new version v2.0.4.
You can now change the generated type by configuring the method `type(fieldType, fieldName, interfaceName)` in your [configuration file](https://github.com/erikvullings/strapi-to-typescript#advanced-configuration)

```javascript
    type: (fieldType, fieldName) => {
      if(fieldName === 'id') return 'number';
    },
```

**Comment by timohofmeijer** [comment]
Works like a charm! Many thanks
